File: DataScience/ControlStructures/1/sum.py
def sumofdigits(n):
    sum = 0
    while n != 0:
        sum += n % 10
        n = n // 10
    print(int(sum))


def startingPoint():
    val = int(input("Enter Number : "))
    sumofdigits(val)


if __name__ == "__main__":
    startingPoint()

File: ArtificialIntelligence/sum/sum.pl
go:-
    write('Enter first number : '),
    read(N1),
    write('Enter second number '),
    read(N2),
    sum(N1,N2,R),
    write('Sum : '),
    write(R).
sum(X,Y,Z):-
    Z is X+Y.